src/pages/basic_data/app_admin/central_import/CentralImportPage.tsx

```diff
@@ -19,10 +19,10 @@ export const CentralImportPage = () => {
   const fromParam = params.get(SearchParam.From);
   const rowsPerPage = (resultsParam && +resultsParam) || ROWS_PER_PAGE_OPTIONS[1];
   const page = (fromParam && resultsParam && Math.floor(+fromParam / rowsPerPage)) || 0;
-
+  const query = '';
   const importCandidateQuery = useQuery({
-    queryKey: ['importCandidates'],
-    queryFn: fetchImportCandidates,
+    queryKey: ['importCandidates', rowsPerPage, page, query],
```

Comment:
Foreslår å fjerne `query`, da den kan lede til at verdien i `queryFn` og `queryKey` kan bli ulik en gang, og den uansett ikke har noen spesiell funksjon per nå
```suggestion
    queryKey: ['importCandidates', rowsPerPage, page, ''],
```